main.css

```diff
@@ -1,33 +1,116 @@
-.card {
-    width: 300px;
+
+body {
+    margin: 0px;
+    box-sizing: border-box;
+    overflow-x: hidden;
+}
+
+footer {
+    background-color: rgba(71, 69, 66, 0.5);
+    padding: 30px;
+    margin: 50px 0 0 0;
+    width: 100%;
+    display: flex;
+    justify-content: space-around;
+}
+
+.footer__copyright {
+    display: inline-block;
+}
+
+header {
+    width: 100%;
+    background-color: black;
+    display: flex;
+    align-items: center;
+    padding: 0 50px;
+    margin-bottom: 30px;
 }
 
-.poster {
-    width: 300px;
+.logo {
+    width: 150px;
 }
 
-.title {
-    font-size: 1.4rem;
-    font-weight: bold;
-    text-align: center;
+.button_home {
+    color: white;
+    display: block;
+    text-decoration: none;
+    padding: 20px;
+    padding-left: 100px;
+    font-family: Arial Black;
+    letter-spacing: 5px;
+    font-size: 25px;
 }
 
-.overview {
-    height: 75px;
+.card {
+    /* width: 230px; */
+    /* height: 550px; */
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-around;
+    align-items: center;
     overflow: hidden;
+    border-radius: 10px;
+    margin: 15px;
+    border: 1px solid rgb(207, 207, 207);
+    transition: width, height, 0.5s;
 }
 
-footer {
-    background-color: rgba(71, 69, 66, 0.5);
-    position: fixed;
-    bottom: 0px;
-    padding: 30px;
-    left: 0px;
+.card:hover {
+    transform: scale(1.1);
+}
+
+.card--img {
     width: 100%;
 }
 
-header {
+.card--movie-name {
+    font-family: 'Fredoka One', cursive;
+    margin: 0;
+    padding: 15px;
+}
+
+.card--movie-overview {
+    font-family: 'Fredoka One', cursive;
+    margin: 0 0 20px 0;
+    padding: 0 25px 25px 25px;
+    height: 110px;
+    overflow: hidden;
+}
+
+.main--movie-list {
+    display: grid;
+    grid-template-columns: repeat(auto-fill, minmax(270px, 1fr));
+    grid-row-gap: 50px;
+    justify-items: center;
+}
+
+.main--movie-info {
+    display: block;
+    position: relative;
+    background-image: url("./images/background.jpg");
+    min-height: calc(100vh - 278px);
+    background-size: cover;
+    background-position: 50%;
+    z-index: -2;
+    color:aliceblue;
+}
+
+.main--movie-info::before {
+    z-index: -1;
+    content: "";
+    position: absolute;
     width: 100%;
-    background-color: rgba(71, 69, 66, 0.5);
-    height: 50px;
+    height: 100%;
+    top: 0;
+    left: 0;
+    background: rgba(0, 0, 0, 0.7);
+}
+
+.footer__movie-detail {
+    margin: 0;
```

Comment:
@Mmarvinn for this will be better to create one atoms class 
```
  .mg-0 { 
    margin: 0; 
  }
```

Reply:
fix.